### ISSUE TITLE
[A11yAudit] Inputs will not rely on the label for placeholder text

### ISSUE THREAD

**Issue by ssabo1** [sparkdesignsystem/Backlog#1338]
**User Story**
As Spark we want to 

**Notes:**
Paciello Audit on Spark Design System
QLC-9
Form controls rely on placeholder text as a label

ISSUE DESCRIPTION
Certain form fields throughout the website are only labeled by a placeholder or rely on a placeholder to convey important information. Text marked up using the placeholder attribute will generally be exposed to assistive technologies, so is announced by screen readers. But for other user groups, relying on placeholder text on its own to label form fields is a problematic approach for the following reasons (as identified by the HTML5 specification):

Placeholder text disappears when a user starts typing. This means that there is no visible label when users are entering text into the input. This is a problem for people with cognitive issues that affect short-term memory, as well as a more general usability issue.
When placeholder text shows an example of input, and does not describe the type of input the input field collects, it is not a reliable label and could cause confusion for some users.

When a visible label is displayed on the page, and it is programmatically associated to the control using the for and id attributes, it is possible to click on the label text to move focus into the related form field; however, when this approach is not taken, the clickable area is reduced, thus reducing usability for people with limited dexterity.

From a usability perspective, some users might think the control has already been pre-filled, and ignore it.
From a technical perspective, support for the attribute is varied across browsers and assistive technologies (particularly legacy versions), so not all users will see the placeholder text.

The date input (samples 26-28) and date picker (samples 29-31) rely on a placeholder to communicate the required date format (e.g. MM/DD/YYYY). Similarly, the phone input (samples 49-51) relies on a placeholder to communicate the required format. Most screen readers announce these but, as described above, the placeholder text disappears when a user starts typing, leaving no visible indication of the required format. Furthermore, as described in Insufficient text color contrast (QLC-3) , the placeholder text lacks sufficient contrast.

RECOMMENDATIONS
Form controls should always have a visible label (with some rare exceptions, such as when there are controls, like submit buttons, immediately adjacent to the control, which arguably act as an implicit label) and, if necessary, instructions.

While it can be appropriate to use the placeholder attribute for short hints intended to aid the user with data entry, it is generally recommended to just keep these hints visible at all times.

**AC:**
- Critical information should not be put in placeholder text. Work with Design to confirm that this can be moved to helper text.

**Comment by ssabo1** [comment]
UX to look into date, phone, & SS

**Comment by ssabo1** [comment]
Confirmed: This has been added to Spark Design System Team Backlog.
